retriever/lib/scripts.py

```diff
@@ -60,7 +60,7 @@ def reload_scripts():
         files = [file for file in os.listdir(search_path)
                  if file[-3:] == ".py" and file[0] != "_" and
                  ('#retriever' in
-                  ' '.join(open(join(search_path, file), 'r', encoding='UTF8').readlines()[:2]).lower())
+                  ' '.join(open(join(search_path, file), 'r', encoding=ENCODING).readlines()[:2]).lower())
```

Comment:
```Python
' '.join(open_fr(join(search_path, file), encode=ENCODING).readlines()[:2]).lower())

```

Reply:
I found the problem when running
![default](https://user-images.githubusercontent.com/44888393/53676231-51894b00-3cda-11e9-9840-8e21b8755fa4.PNG)

I think you mean `open_fr(join(search_path, file), encoding=ENCODING) `because the encode should be either True or False in open_fr.
And `' '.join(open_fr(join(search_path, file), encoding=ENCODING).readlines()[:2]).lower())` can give the correct result.
![default](https://user-images.githubusercontent.com/44888393/53676243-7b427200-3cda-11e9-9607-54e7b869f6f1.PNG)


Actually, I think `' '.join(open_fr(join(search_path, file)).readlines()[:2]).lower())` is just fine because the default value for encoding in open_fr is ENCODING.